pkgs/development/r-modules/default.nix

```diff
@@ -11,6 +11,13 @@ let
     inherit (pkgs) gettext gfortran;
   };
 
+  # Set the correct string to download either the binary for
+  # the cpu version of the torch package, or the
+  # the cuda-enabled version.
+  # To use gpu acceleration, set `config.cudaSupport = true;`
+  # when importing nixpkgs in your shell
+  accel = if pkgs.config.cudaSupport then "cu118" else "cpu";
```

Comment:
Could we move this inside the torch `let` statement like this:

```
    torch = let
       accel = if pkgs.config.cudaSupport then "cu118" else "cpu";
       binary_sha = if pkgs.config.cudaSupport then
         "sha256-a80sG89C0svZzkjNRpY0rTR2P1JdvKAbWDGIIghsv2Y=" else
         "sha256-qUn8Rot6ME7iTvtNd52iw3ebqMnpLz7kwl/9GoPHD+I=";
      in
```

Separating them will make maintenance a bit harder, and no other package is going to need the strings "cu118" vs "cpu".

Reply:
done!